django-stubs/test/runner.pyi

```diff
@@ -36,14 +36,18 @@ class DebugSQLTextTestResult(TextTestResult):
     def addError(self, test: Any, err: Any) -> None: ...
     def addFailure(self, test: Any, err: Any) -> None: ...
 
-class PDBDebugResult(TextTestResult): ...
+class PDBDebugResult(TextTestResult):
+    def debug(self, error: Any) -> None: ...
+
+class DummyList:
+    def append(self, item: Any) -> None: ...
 
 class RemoteTestResult:
     events: list[Any]
     failfast: bool
     shouldStop: bool
     testsRun: int
-    def __init__(self) -> None: ...
+    def __init__(self, *args: Any, **kwargs: Any) -> None: ...
```

Comment:
Can you please add a TODO entry to sync this signature with `unittest.TestResult`?